src/explore-education-statistics-admin/src/pages/release/edit-release/content/ReleaseContext.tsx

```diff
@@ -0,0 +1,237 @@
+import produce from 'immer';
+import {
+  EditableContentBlock,
+  ExtendedComment,
+  EditableRelease,
+} from '@admin/services/publicationService';
+import { DataBlock } from '@common/services/dataBlockService';
+import { ContentSection } from '@common/services/publicationService';
+import React, { createContext, useContext, useReducer } from 'react';
+import ReleaseDispatchAction from './actions';
+
+type Dispatch = (action: ReleaseDispatchAction) => void;
+type State = {
+  release: EditableRelease | undefined;
```

Comment:
Shouldn't we try and guarantee that this should be defined when we set up our context provider?